Modules/ThirdParty/MetaIO/src/MetaIO/src/metaTube.h

```diff
@@ -117,9 +117,9 @@ class METAIO_EXPORT MetaTube : public MetaObject
 
     void PrintInfo(void) const override;
 
-    void CopyInfo(const MetaObject * _object) override;
+    void CopyInfo(const MetaTube * _object);
 
-    void Clear(void) override;
+    void Clear(void);
```

Comment:
The `override` specifier that is removed here does seem useful to me!

Reply:
Agreed.  This probably needs to be added again in the upstream version.

Approving to not let perfection be the enemy of the good.

Reply:
I think the removal of this override will cause compiler warnings.

Reply:
I think @aylward removed it. Maybe he should pitch in?